api/envoy/config/ratelimit/v3/rls.proto

```diff
@@ -31,4 +32,8 @@ message RateLimitServiceConfig {
   // API version for rate limit transport protocol. This describes the rate limit gRPC endpoint and
   // version of messages used on the wire.
   core.v3.ApiVersion transport_api_version = 4 [(validate.rules).enum = {defined_only: true}];
+
+  // Specifies whether the RPC communication with RLS server should be unary or streaming RPC.
```

Comment:
Add default value to the comment.
If the default value might be changed in the future, then you should consider the Bool WKT.

Reply:
The bool default value in proto is false(https://developers.google.com/protocol-buffers/docs/proto3#default), which is intention here. 
I guess you mean add this info to comment? Done.